apps/call.go

```diff
@@ -149,14 +150,40 @@ func NewProxyCallResponse(response CallResponse, metadata *AppMetadataForClient)
 	}
 }
 
-func NewErrorCallResponse(err error) CallResponse {
+func NewErrorResponse(err error) CallResponse {
 	return CallResponse{
 		Type: CallResponseTypeError,
 		// TODO <>/<> ticket use MD instead of ErrorText
 		ErrorText: err.Error(),
 	}
 }
 
+func NewOKResponse(data interface{}, textArgs ...interface{}) CallResponse {
```

Comment:
0/5 Does it makes sense to mimic `fmt.Printf` here and use `string` as the first parameter?
```suggestion
func NewOKResponse(data interface{}, format string, a ...interface{}) CallResponse {
```

Reply:
There are 2 fields to fill out, `Data` and `Markdown`. 0/5 maybe just have `NewDataResponse(interface{})` and `NewMarkdownResponse(fmt, ...)` and require mixed instances to be created explicitly?